src/gui/accountsettings.cpp

```diff
@@ -488,6 +488,10 @@ void AccountSettings::slotAccountStateChanged()
             icon = StatusIcon::Warning;
         }
         showConnectionLabel(tr("Connected"), icon, errors);
+        if (accountsState()->supportsSpaces()) {
+            connect(accountsState()->account()->spacesManager(), &GraphApi::SpacesManager::updated, this, &AccountSettings::slotSpacesUpdated);
```

Comment:
Won't that result in a double connect? https://github.com/owncloud/client/blob/460e3229af19072550423e44bde1dc0a06dffcab/src/gui/accountsettings.cpp#L80-L80